data_preparation/predictor_data_prep.r

```diff
@@ -0,0 +1,383 @@
+if(!"terra" %in% installed.packages()){install.packages("terra")}
+if(!"reshape2" %in% installed.packages()){install.packages("reshape2")}
+library(terra)
+library(reshape2)
+library(stringr)
+
+valid_fpar_values <- function(fpar_rast) {
+  # check data_preparation/global_fpar_500m
+  # FPAR values: 0 to 100 (%). Valid range is 0-100.
+  # Flags: 251 is "other land", 254 is "water", 255 is "not processed".
+  fpar_rast[fpar_rast > 100] <- NA
+  fpar_rast[fpar_rast <= 0] <- NA
```

Comment:
@krsnapaudel fpar 's valid range includes 0

Reply:
Yes, but zero does not add to much when multiplied or summed.